gcc/rust/lex/rust-lex.h

```diff
@@ -362,8 +364,7 @@ class Lexer
     {
       if (offs >= buffer.size ())
 	return EOF;
-
-      return buffer.at (offs++);
+      return (uint8_t) buffer.at (offs++);
     }
```

Comment:
Added casting to prevend bytes whose MSB is 1 from being sign-extended.
Without casting , for example, `0xfe` becomes `0xfffffffe`.

Reply:
bugfix too